### PR TITLE
Fix index-64bits.js

### PR DESCRIPTION
**Pull request by klahnakoski**
Here is a proposed change; it simplifies the organization logic, combines the logic to one page, and looks a bit nicer

### CODE REVIEW

**Comment by klahnakoski** [comment]
@armenzg I do not know how to inject variables into the test for the page, so it seems to default to 32 bits.

What do we use to run the test suite in the debugger so we can trace the variables it is injecting into our code?

**Comment by klahnakoski** [comment]
@armenzg nm, it seems I bumbled through fine enough.

**Comment by klahnakoski** [comment]
The diff is much less frightening with white space ignored (`?w=1`)

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/254/files?w=1

File: src/containers/PerfherderGraphContainer/index.jsx [hunk 1]
```diff
@@ -7,9 +7,11 @@ import getPerferherderData from '../../utils/perfherder/chartJs/getPerfherderDat
 
 const styles = () => ({
   title: {
-    color: 'white',
-    backgroundColor: 'black',
-    padding: '.2rem .3rem .3rem .3rem',
+    color: '#56565a',
+    fontSize: '1rem',
+    backgroundColor: '#d1d2d3',
+    padding: '0 0 0 0.1rem',
```
[the remark this answers is not in the document]
Comment (klahnakoski) [reply src/containers/PerfherderGraphContainer/index.jsx]: Yes, I was attempting to unify the overall look of the page: With the chart headers having no padding:

image


I will  revert the padding change

**Comment by klahnakoski** [comment]
@armenzg 

Changes made (or Issues created)  

I fixed `?full` mode, and added a link to tp6 (with four pages as a preview)

**Comment by klahnakoski** [comment]
see 

https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/254/commits/dca8e3889d4be56c1348ac1d9a9c0614bcf4eb55?w=1

File: src/utils/componentEnhancers.jsx [hunk 2]
```diff
@@ -4,9 +4,9 @@ import GenericErrorBoundary from '../components/genericErrorBoundary';
 const wrapSectionComponentsWithErrorBoundaries = (sections = []) => (
   sections.map(section => ({
     ...section,
-    rows: section.rows.map(row => row.map((component, index) => (
+    rows: section.rows.map((component, index) => (
       <GenericErrorBoundary key={index}>{component}</GenericErrorBoundary>
```
[the remark this answers is not in the document]
Comment (armenzg) [reply src/utils/componentEnhancers.jsx]: Filed
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/258

**Pull request merged**